gait_state:
    # Location the feet return to at the end of stride phase, in the ground frame, meters
    # These magic numbers are the direction from the body to the shoulder_fixed frames, extended
    # out the lengths of the coxa + femur past the hip joint
    # Coordinates (xyz) x Leg
    foot_center_init: [ 1.76142685e-01,  1.76142685e-01,             0.0,             0.0, -1.76142685e-01, -1.76142685e-01,
                        1.27200542e-01, -1.27200542e-01,  2.06701000e-01, -2.06701000e-01,  1.27200542e-01, -1.27200542e-01,
                                   0.0,             0.0,             0.0,             0.0,             0.0,             0.0 ]

    max_stride_length: 0.08 # meters
